README.md

```diff
@@ -1,21 +1,13 @@
 # TicTacToe
-
-**TODO: Add description**
-
-## Installation
-
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `tic_tac_toe` to your list of dependencies in `mix.exs`:
-
-```elixir
-def deps do
-  [
-    {:tic_tac_toe, "~> 0.1.0"}
-  ]
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/tic_tac_toe](https://hexdocs.pm/tic_tac_toe).
+Command line Tic-Tac-Toe game
+ 
+## Play the game
+1. [Install Elixir](https://elixir-lang.org/install.html)
```

Comment:
Can you specify the version of Elixir?

Reply:
Is that you Eric?  Sure.

Reply:
That's me - I promise you aren't getting random code reviews.